roles/lib_openshift/src/ansible/oc_secret.py

```diff
@@ -0,0 +1,37 @@
+# pylint: skip-file
+# flake8: noqa
+
+
+def main():
+    '''
+    ansible oc module for managing OpenShift Secrets
+    '''
+
+    module = AnsibleModule(
+        argument_spec=dict(
+            kubeconfig=dict(default='/etc/origin/master/admin.kubeconfig', type='str'),
+            state=dict(default='present', type='str',
```

Comment:
The `DOCUMENTATION` for the `state` key says

> required: true
> default: present

but this parameter is not defined with `required=True`. As stated in the other review comment, if only `present` is supported, why provide a list of `choices`?